includes/wizards/class-setup-wizard.php

```diff
@@ -0,0 +1,144 @@
+<?php
+/**
+ * Newspack setup wizard. Required plugins, introduction, and data collection.
+ *
+ * @package Newspack
+ */
+
+namespace Newspack;
+
+use \WP_Error, WP_REST_Server;
+defined( 'ABSPATH' ) || exit;
+require_once NEWSPACK_ABSPATH . '/includes/wizards/class-wizard.php';
+
+define( 'NEWSPACK_SETUP_COMPLETE', 'newspack_setup_complete' );
+
+/**
+ * Setup Newspack.
+ */
+class Setup_Wizard extends Wizard {
+	/**
+	 * The slug of this wizard.
+	 *
+	 * @var string
+	 */
+	protected $slug = 'newspack-setup-wizard';
+	/**
+	 * The capability required to access this wizard.
+	 *
+	 * @var string
+	 */
+	protected $capability = 'edit_products';
```

Comment:
Probably want `install_plugins` and/or `manage_options` instead of `edit_products`.

Reply:
Good call: https://github.com/Automattic/newspack-plugin/pull/114/commits/74493922971504fc6879cd64ac9aba0bc11d4e73